torch/distributions/utils.py

```diff
@@ -166,6 +166,12 @@ def probs_to_logits(probs, is_binary=False):
     return torch.log(ps_clamped)
 
 
+def set_default_validate_args(value):
```

Comment:
I think this might be cleaner as a `@staticmethod` of the `Distribution` class as in #133, but I defer to your and Adam's jugement.

Reply:
Ok. I will do that. Do you guys want me to add #133 's changes in the new PR?
cc @vishwakftw